docs/wiki/ConsumerGuide.md

```diff
@@ -0,0 +1,63 @@
+
+- Clone this repository
+- Fork & Clone this repository
+- Download a `.zip` copy of this repo
+- Upload a copy of the locally cloned/downloaded modules to your own:
+  - Git Repository
+  - Private Bicep Module Registry
+    - See:
+      - [Azure Landing Zones - Private/Organizational Azure Container Registry Deployment (also known as private registry for Bicep modules)](https://github.com/Azure/ALZ-Bicep/wiki/ACRDeployment)
+      - [Create private registry for Bicep modules](https://docs.microsoft.com/azure/azure-resource-manager/bicep/private-module-registry)
+  - Template Specs
+    - See:
+      - [Azure Resource Manager template specs in Bicep](https://docs.microsoft.com/azure/azure-resource-manager/bicep/template-specs)
+- Use and reference the modules directly from the Microsoft Public Bicep Registry - ***Coming Soon (awaiting feature release in Bicep)***
+
+The option to use will be different per consumer based on their experience and skill levels with the various pieces of technology and their features.
+
+> Consumers can also use the above listed methods together with CI/CD pipelines to deploy the modules. See [Azure Landing Zones Bicep - Pipelines](https://github.com/Azure/ALZ-Bicep/wiki/PipelinesOverview) for more information and guidance
+
+## Customizing the `ALZ-Bicep` Modules
+
+Whatever way you may choose to consume the modules we do do expect, and want, customers and partners to customize the modules to suit their needs and requirements for their design in their local copies of the modules.
+
+For example, if you want to change the names of the Management Groups, outside of what the parameters allow you to change and customize, then by opening up the `managementGroups.bicep` module you should be able to read, understand and customize the required lines to meet your requirements easily; due to the way the modules have been authored as shared above.
+
+This customized module can then be deployed into your environment to deliver the desired changes and architecture.
+
+<!-- markdownlint-disable -->
+> **IMPORTANT:** If you believe the changes you have made should be more easily available to be customized by a parameter etc. in the module, then please raise an [issue](https://github.com/Azure/ALZ-Bicep/issues) for a 'Feature Request' on the repository 👍
+> 
+> If you wish to, also feel free to submit a pull request relating to the issue which we can review and work with you to potentially implement the suggestion/feature request 👍
+<!-- markdownlint-restore -->
+
+## Considerations when Customizing the `ALZ-Bicep` Module
+
+Whilst customizing the modules on the whole is straightforward and simple, there are some key things to consider depending on which modules you customize.
+
+> See something missing from here? Please raise an [issue](https://github.com/Azure/ALZ-Bicep/issues) on the repo and let us know and feel free to contribute a pull request also 👍
```

Comment:
Not sure if this is a typo or related to the above sentence.

Reply:
Nope this is valid here as want to know if customers think anything else should be added in this section